Server/controllers/controllerUtils.js

```
@@ -0,0 +1,19 @@
+const handleValidationError = (error, serviceName) => {
+  error.status = 422;
+  error.service = serviceName;
+  error.message =
+    error.details?.[0]?.message || error.message || "Validation Error";
+  return error;
+};
+
+const handleError = (error, serviceName, method, code = 500) => {
+  error.code === undefined ? (error.code = code) : null;
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)